src/common/functions/functions.sh

```diff
@@ -77,6 +80,21 @@ function check_root {
   fi;
 }
 
+function check_existing_installation {
+  # check existing installation if -m argument (minimal upgrade) is detected
+  for i in ${run_args[@]}; do
+    if [[ $i == '-m' ]]; then
+      if [ -f "${binary_path}" ]; then
```

Comment:
we should be able to use `[[ ! -f "${binary_path}" ]];`, and avoid needing a `break;`